k8s/test/segment_server_service.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: segment-server-lb
+  namespace: chroma
+spec:
+  ports:
+    - name: segment-server-port
+      port: 50052
```

Comment:
50052 or 50051? Not sure it should be different, just double-checking

Reply:
i left it like this so in local tests it doesn't conflict, we can change it back in future - these manifests are temp anyways